package.json

```diff
@@ -23,8 +23,8 @@
     "packages/browser",
     "packages/core",
     "packages/ember",
-    "packages/eslint-config-sentry-sdks",
-    "packages/eslint-plugin-sentry-sdks",
+    "packages/eslint-config",
+    "packages/eslint-plugin",
```

Comment:
Let's keep the `-sentry-sdks` since we will most likely have more eslint packages